bgpd/bgp_nht.c

```diff
@@ -137,6 +138,7 @@ static int bgp_isvalid_nexthop_for_mpls(struct bgp_nexthop_cache *bnc,
 	 * Transit routes for SRv6-VPN are valid.
 	 * - Otherwise check for mpls-gre acceptance
 	 */
+
```

Comment:
Intentional change? Can we reduce such changes everywhere in this patch?

Reply:
ok